bpaotu/bpaotu/frontend/src/pages/search_page/index.tsx

```diff
@@ -30,18 +31,21 @@ export const SearchPage = props => (
     </Row>
 
     <Row className="space-above">
-      <Col sm={{ size: 2, offset: 5 }}>
-        <SearchButton isDisabled={props.isSearchButtonDisabled} search={props.search} />
-      </Col>
+      {props.isSearchButtonDisabled
+        ? <Col sm={{ size: 2, offset: 6 }}> <LoadingSpinner /> </Col>
+        : <Col sm={{ size: 2, offset: 5 }}> <SearchButton isDisabled={props.isSearchButtonDisabled} search={props.search} /> </Col>
```

Comment:
you can drop the `isDisabled` property now because the button will not be displayed at all in that case.